build.psm1

```diff
@@ -2808,6 +2808,25 @@ assembly
     }
 }
 
+$psOptionsPath = 'psoptions.json'
```

Comment:
Should we exclude the file from git?

Reply:
We will only use this during CI workflow currently.  We can ignore it if it becomes a problem.